shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -328,16 +327,24 @@ class JSBackend {
       JSConstDecl(defResultObjName, JSRecord(Nil)) ::
         JSConstDecl(exprResultObjName, JSArray(Nil)) ::
         typeDefs
-          .map { case TypeDef(kind, Primitive(name), typeParams, actualType, _, _) => // TODO: handle methods
-            kind match {
-              case Cls =>
-                classNames += name
-                val cls = translateClassDeclaration(name, actualType)
-                nameClsMap += name -> cls
-                cls
-              case Trt => JSComment(s"trait $name")
-              case Als => JSComment(s"type alias $name")
-            }
+          .map {
+            case TypeDef(
+                  kind,
+                  Primitive(name),
+                  typeParams,
+                  actualType,
+                  _,
+                  _
+                ) => // TODO: handle methods
```

Comment:
This comment caused the entire block to be refactored in an awkward way... 😅

Reply:
😅 Yes. I merged an old `mlscript` branch. Now this version of `mlscript` branch was merged.